.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,40 @@
+---
+name: 🐞 Bug
+about: File a bug report
+title: '[BUG] <title>'
+labels: bug, triage
+assignees: ''
+
+---
+
+<!--
+Note: Please search to see if an issue already exists for the bug you encountered.
+-->
+
+### Current Behavior:
+<!-- A concise description of what you're experiencing. -->
+
+### Expected Behavior:
+<!-- A concise description of what you expected to happen. -->
+
+### Steps To Reproduce:
+<!--
+Example: steps to reproduce the behavior:
+1. In this environment...
+1. With this config...
+1. Run '...'
+1. See error...
```

Comment:
Should these be 1, 2, 3, 4 or bullets?

Reply:
Probably makes more sense (its not going to be rendered, its just a comment). Again this template is just a copy of the standard one github recommends so I suppose it works either way.